src/components/NftRewards/NftTierCard.tsx

```diff
@@ -55,11 +55,13 @@ export function NftTierCard({
     setPreviewVisible(true)
   }
 
-  const remainingSupplyText =
-    rewardTier?.remainingSupply !== undefined &&
-    rewardTier.remainingSupply < MAX_REMAINING_SUPPLY
-      ? t`${rewardTier?.remainingSupply} remaining`
-      : t`Unlimited`
+  const remainingSupply = 0 //rewardTier?.remainingSupply
+  const canSelect = remainingSupply && remainingSupply > 0
+  const remainingSupplyText = !canSelect
+    ? t`SOLD OUT`
+    : remainingSupply === MAX_REMAINING_SUPPLY
+    ? t`Unlimited`
```

Comment:
**issue** i don't think this is true. Even when a tier uses `MAX_REMAINING_SUPPLY`, the supply still decrements pretty sure, so as soon as one NFT of that tier is sold, remainingSupply === MAX_REMAINING_SUPPLY will never be true.

Wouldn't the idea be to have `initialSupply === MAX_REMAINING_SUPPLY` or somethin?

Reply:
Hmm yeah actually not sure about that, good pick up. I've changed it to use the `maxSupply` instead of remaining just in case 👍